ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -51,6 +51,7 @@ promotion:
   namespace: ocp
   to:
   - additional_images:
+      ci-image: ci-image
```

Comment:
give this an olm specific name?

Reply:
it's promoted to an olm-specific namespace